### PR TITLE
Sponsor tracking links

### PR DESCRIPTION
**Pull request by psiinon**
I did think about creating a shortcut so links could be managed in one place but it seemed like too much work for now :wink:

### CODE REVIEW

File: site/content/sponsor.md [hunk 1]
```diff
@@ -48,6 +48,7 @@ we want to make it straightforward and as mutually beneficial as possible.
 * Name and link on any sponsored [Roadmap](/docs/roadmap) items.
 * Priority placement on [Third Party Products and Services](/third-party-services/) page.
 * Priority placement on [Success Stories](/success/) page.
+* Tracking links option.
```
Comment (kingthorin) [inline site/content/sponsor.md:51]:
```suggestion
* Analytic links option.
```

File: site/content/sponsor.md [hunk 2]
```diff
@@ -63,6 +64,7 @@ we want to make it straightforward and as mutually beneficial as possible.
...
```
Comment (kingthorin) [inline site/content/sponsor.md:67]:
```suggestion
* Analytic links option.
```

**Review by kingthorin — APPROVED** [review]

File: site/content/sponsor.md (hunk 2 above)
Comment (psiinon) [reply site/content/sponsor.md]: +1 changed 😁

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**